wolfcrypt/benchmark/benchmark.c

```diff
@@ -3855,9 +3865,13 @@ static void bench_rsa_helper(int doAsync, RsaKey rsaKey[BENCH_MAX_PENDING],
     double      start = 0.0f;
     const char**desc = bench_desc_words[lng_index];
 
+    if (RSA_KEY_SZ != rsaKeySz) {
```

Comment:
Should this be updated to allow for the case were rsaKeySz is less? Example: `if (rsaKeySz  > RSA_KEY_SZ) {`. Or maybe just use `RSA_BUF_SIZE` and remove the section added for `RSA_KEY_SZ`?

Reply:
I think your suggestion for RSA_BUF_SIZE might be the cleanest.